frontend/components/Header.tsx

```diff
@@ -42,7 +48,7 @@ const Header = () => {
           </Link>
         </div>
         <div className="flex items-center justify-end space-x-2">
-          <WalletMultiButton className="primary-btn pt-0.5" />
```

Comment:
this was causing hydration error when wallet is connected because client side and server side was returning different code